src/components/StoreList/StoreList.js

```diff
@@ -2,16 +2,19 @@ import React from 'react';
 import ButtonList from '../ButtonList/ButtonList';
 import {sendSlackMessage, getUserSlackID} from '../../utils/slack';
 import ErrorMessage from '../ErrorMessage/ErrorMessage';
+import PropTypes from 'prop-types';
 
 const StoreList = props => {
 	return (
 		<div>
+			Please select the correct item:
 			<ButtonList
-				items={props.storeList}
+				items={Object.values(props.storeList)}
```

Comment:
Could we do this `Object.values` call in a selector/mapStateToProps?

This allows the component to not have to care about how it's data is mapped. Also has the benefit that this isn't being recomputed every time any of it's parents update (causing a rerender)